internal/network/manifests_generator.go

```diff
@@ -238,6 +239,126 @@ func (m *ManifestsGenerator) AddSchedulableMastersManifest(ctx context.Context,
 	return nil
 }
 
+const (
+	diskEncryptionManifest = `apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  name: {{.NAME}}
```

Comment:
You don't define this key in manifestParams

Reply:
Missed that - good catch.

Reply:
Fixed.